resources/js/src/app/mixins/url.js

```diff
@@ -155,7 +155,7 @@ export default {
                     min: facetValue.min,
                     max: facetValue.max
                 };
-            } else if (this.facet.select === 'single' && facetId !== 'cat') {
+            } else if ((this.facet.select === 'single' && facetId !== 'cat') || (this.facet.type === 'select' && this.facet.select === 'multiple')) {
```

Comment:
Thought about adding the case for dropdown here. Only not sure because of select=multiple . Seems like it should be multiple selection not single.